release notes/v0.48.0.md

```diff
@@ -33,8 +33,7 @@ _Format as `<number> <present_verb> <object>. <credit>`_:
 
 ## Bug fixes
 
-_Format as `<number> <present_verb> <object>. <credit>`_:
-- _`#111` fixes race condition in runtime_
+- [browser#1042](https://github.com/grafana/xk6-browser/pull/1042), [browser#1069](https://github.com/grafana/xk6-browser/pull/1069) Fixes `browserContext.waitForEvent`. This fix involved promisifying the `waitForEvent` API as well as various internal refactors.
```

Comment:
Is this a breaking change? 🤔 Should we move it to breaking changes if so?

Reply:
Good point, i guess it could go in breaking changes since we're making it async now. I think though, that this feature never really worked even when it was synchronous (which is likely a sign that maybe it's not that important), so I still think it's better left where it is. WDYT?

Reply:
Yep, I'm in between, too 😒